arlo-client/src/__snapshots__/App.test.tsx.snap

```diff
@@ -40,71 +40,6 @@ exports[`App renders properly 1`] = `
         </div>
       </div>
     </div>
-    <div
-      class="sc-jbKcbu dZqbbS"
-    >
-      <img
-        alt="Arlo, by VotingWorks"
-        height="50px"
-        src="/arlo.png"
-      />
-      <div
-        class="sc-gzVnrw coBAaV"
-      >
-        <label
-          for="audit-name"
-          id="audit-name-label"
-        >
-          Give your new audit a unique name.
-          <div
-            class="sc-jqCOkK diELAR sc-gqjmRU egaEhI"
-          >
-            <div
-              class="bp3-input-group sc-VigVT hfLuvB"
-            >
-              <input
-                aria-labelledby="audit-name-label"
-                class="bp3-input"
-                id="audit-name"
-                name="auditName"
-                style="padding-right: 10px;"
-                value=""
-              />
-            </div>
-          </div>
-        </label>
-      </div>
-      <button
-        class="bp3-button bp3-fill bp3-large bp3-intent-primary sc-hqyNC sUzDH sc-bwzfXH eBfcXh"
-        type="button"
-      >
-        <span
-          class="bp3-button-text"
-        >
-          Create a New Audit
-        </span>
-      </button>
-      <button
-        class="bp3-button bp3-fill bp3-large bp3-intent-primary sc-hqyNC sUzDH sc-bdVaJa gLVjYw"
-        type="button"
-      >
-        <span
-          class="bp3-button-text"
-        >
-          Log in as an Audit Admin
-        </span>
-      </button>
-      <button
-        class="bp3-button bp3-fill bp3-large bp3-intent-primary sc-hqyNC sUzDH sc-bdVaJa gLVjYw"
-        type="button"
-      >
-        <span
-          class="bp3-button-text"
-        >
-          Log in as a Jurisdiction Admin
-        </span>
-      </button>
-    </div>
```

Comment:
This snapshot change looks suspicious. Do we not expect any of this to render anymore?

Reply:
@jonahkagan It does look a bit suspicious. The create new audit button isn't supposed to be there, but the log in ones should still be there. I'm not super concerned about it yet, though. This PR is mainly just file reorganization with a few simple test fixes. I'm planning another one for updating more of the api stuff, and a few others. The snapshots will likely not get completely fixed until last, once everything else is done moving around. These updates are mostly due to me moving tests around in different files, with some side effects, I think.